package NGranet.NProto;

message TGranetDomain {
    // ELanguage
    optional uint32 Language            = 1;
    optional bool IsPASkills            = 2;
    optional bool IsWizard              = 3;
    reserved 4;
    reserved "IsBeggins";
    optional bool IsSnezhana            = 5;
}
